powerauth-webflow-client/pom.xml

```diff
@@ -86,13 +86,6 @@
             <artifactId>powerauth-nextstep-client</artifactId>
         </dependency>
 
-        <!-- H2 DB -->
-        <dependency>
-            <groupId>org.hsqldb</groupId>
-            <artifactId>hsqldb</artifactId>
```

Comment:
hsqldb is not h2, if needed for the demo in the future, should be enabled only for a specific profile.